src/app/features/attendance/analysis/attendance-details/attendance-details.stories.ts

```
@@ -47,7 +48,9 @@ const activityAttendance = ActivityAttendance.create(new Date("2020-01-01"), [
     new Date("2020-01-04"),
   ),
 ]);
-activityAttendance.events.forEach((e) => (e.subject = demoActivity.title));
+activityAttendance.events.forEach(
```

Biome (2.4.4)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)